### ISSUE TITLE
Where to download pth files?

### ISSUE THREAD

**Issue by lizhiwen19900709** [hustvl/Symphonies#1]
Where to download maskdino_r50_50e_300q_panoptic_pq53.0.pth  files?

**Comment by wondervictor** [comment]
Hi @lizhiwen19900709, thanks for your interest in Symphonies. The checkpoint of MaskDINO can be found at [MaskDINO](https://github.com/IDEA-Research/detrex-storage/releases/download/maskdino-v0.1.0/maskdino_r50_50ep_300q_hid2048_3sd1_panoptic_pq53.0.pth).

**Comment by npurson** [comment]
Thanks for raising this question. As stated in `maskdino/README.md`, the mmdet implementation of MaskDINO is based on the code provided in https://github.com/open-mmlab/mmdetection/pull/9808.

To obtain the pretrained weights, you can use the [conversion scripts](https://github.com/Li-Qingyun/mmdetection/tree/add-mask-dino/projects/MaskDINO/model_converters) provided in the repo to convert the weights from the [original MaskDINO repo](https://github.com/IDEA-Research/MaskDINO) into the mmdet format.

Alternatively, for usage convenience, we may also consider uploading the converted weights to our repository later. Please let us know if we can be of any assistance.

**Comment by npurson** [comment]
> Thanks for raising this question. As stated in `maskdino/README.md`, the mmdet implementation of MaskDINO is based on the code provided in [open-mmlab/mmdetection#9808](https://github.com/open-mmlab/mmdetection/pull/9808).
> 
> To obtain the pretrained weights, you can use the [conversion scripts](https://github.com/Li-Qingyun/mmdetection/tree/add-mask-dino/projects/MaskDINO/model_converters) provided in the repo to convert the weights from the [original MaskDINO repo](https://github.com/IDEA-Research/MaskDINO) into the mmdet format.
> 
> Alternatively, for usage convenience, we may also consider uploading the converted weights to our repository later. Please let us know if we can be of any assistance.

The converted pre-trained weight has been uploaded to [here](https://github.com/hustvl/Symphonies/releases/download/v1.0/maskdino_r50_50e_300q_panoptic_pq53.0.pth).
